### ISSUE TITLE
location var in README usage example

### ISSUE THREAD

**Issue by doesntgolf** [mozilla/readability#272]
In the [usage example](https://github.com/mozilla/readability#usage) in the README, the initial `var location = document.location;` causes a refresh. I just had a very confusing few minutes :)

**Comment by leibovic** [comment]
That's strange, why does that cause a refresh? I can't reproduce that locally if I try running that line in the web console.

**Comment by gijsk** [comment]
> That's strange, why does that cause a refresh? I can't reproduce that locally if I try running that line in the web console.

That would be because the web console has its own scope. Try: http://output.jsbin.com/fifavohogi ( see http://jsbin.com/fifavohogi/edit?html,js for code) .

**Comment by gijsk** [comment]
Fixed by using `loc` instead. @leibovic , I figured this was trivial enough not to need review, but let me know if you want to look at it in more detail. As to the "why does that cause a refresh" - variables in the global scope are written onto `window`, and assigning to `window.location` has similar semantics as assigning to `window.location.href`. At least, that's my 20-second explanation, I'd have to doublecheck the spec to give a more precise answer. Using a different variable name avoids this issue. :-)
